Client/Frontend/Browser/PrivacyProtection/PrivacyProtection.swift

```diff
@@ -8,8 +8,6 @@ import UIKit
 final class PrivacyProtection: PrivacyProtectionProtocol {
 
     var nonPersistent: Bool {
-        let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
-        return isPrivateBrowsing
+        return true
```

Comment:
Can simplify this to `let nonPersistent = true` if we are never changing it

Reply:
Can we actually just completely remove `PrivacyProtectionProtocol`? The only difference between the two is a `true`/`false` boolean. Can't we just put a flag `nonPersistent` directly onto the webview rather than using a protocol and two custom classes?